testing/integration/alpine/Dockerfile

```diff
@@ -2,6 +2,6 @@
 # Alpine has an up to date NGINX image so use that
 #
 
-ARG NGINX_DEPLOY_VERSION
-FROM nginx:$NGINX_DEPLOY_VERSION-alpine
+ARG NGINX_VERSION
```

Comment:
Most of this is just a but of renaming of test resources to make tests easier to run.